index.html

```diff
@@ -2645,6 +2657,164 @@ <h3>Refreshing</h3>
 
       </section>
 
+      <section>
+        <h3>Rendering</h3>
+
+        <p class="issue" title="FEATURE AT RISK: Rendering">
+The rendering feature might not be included in the final version 2.0
+specification and is thus considered a feature at risk. It is unknown at this
+time if there will be enough implementations for the feature. The Working
+Group is seeking implementer feedback on whether or not they see this
+feature as worth keeping in the final version 2.0 specification.
+        </p>
+
+        <p>
+Rendering hints can be used when the <a>issuer</a> has a specific way that
+they want to express a <a>verifiable credential</a> to an observer through
+a visual, auditory, or haptic mechanism. For example, an <a>issuer</a> of an
+employee badge credential might want to include rich imagery of their corporate
+logo and specific placement of employee information in specific areas of the
+badge. They might also want to provide an audio read out of the important
+aspects of the badge for individuals that have accessibility needs related
+to their eyesight.
+        </p>
+
+        <dl>
+          <dt><var>render</var></dt>
+          <dd>
+The value of the <code>render</code> <a>property</a> MUST specify one or
+more rendering hints that can be used by software to express the
+<a>verifiable credential</a> using a visual, auditory, or haptic mechanism.
+Each <code>render</code> value MUST specify its <a>type</a>, for example,
+<code>SvgRenderingTemplate2023</code>. The precise contents of each
+rendering hint is determined by the specific <code>render</code> <a>type</a>
+definition.
+          </dd>
+        </dl>
+
+        <section>
+          <h4>SvgRenderingTemplate2023</h4>
+
+          <p>
+When an <a>issuer</a> desires to specify SVG rendering instructions for a
+<a>verifiable credential</a>, they MAY add a `render` property that uses the
+data model described below.
+          </p>
+
+          <table class="simple">
+            <thead>
+              <tr>
+                <th style="white-space: nowrap">Property</th>
+                <th>Description</th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <td>id</td>
+                <td>
+A URL that dereferences to an SVG image [[SVG]] with an associated
+`image/svg+xml` media type.
+                </td>
+              </tr>
+              <tr>
+                <td>type</td>
+                <td>
+The <code>type</code> property MUST be <code>SvgRenderingTemplate2023</code>.
+                </td>
+              </tr>
+              <tr>
+                <td>digestMultibase</td>
```

Comment:
... its ok, its non normative right?

Reply:
I don't understand the question. Are you saying "This is ok for now?" or are you asking "Is this a non-normative statement?" -- if so, no, it's a normative statement. Also note the issue marker stating that this is a discussion to be had about the data format.